.eslintrc.js

```diff
@@ -1,12 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
+    "no-proto": 0,
```

Comment:
The `no-proto` rule is set to 0, which disables it. While this might be intentional, it's generally recommended to avoid using `__proto__` due to potential security and performance issues. Consider enabling this rule unless you have a specific reason to disable it.